test/hot_recharge_test.dart

```diff
@@ -106,4 +106,14 @@ void main() async {
       expect(bal, isInstanceOf<ApiResponse>());
     });
   });
+
+  test('End-user Airtime Balance : return ApiResponse instance', () async {
```

Comment:
lets group all `zesa` test into one group and all `airtime` related tests into their own groups defined there
But good work bro. You have done great just those little changes and we will have a uniform development phase